owlbot.py

```diff
@@ -64,7 +64,7 @@
 
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library(source_location='build/src')
-s.copy(templates, excludes=[".kokoro/samples-test.sh", ".kokoro/trampoline_v2.sh", ".github/release-trigger.yml", ".github/sync-repo-settings.yaml"])
+s.copy(templates, excludes=[".kokoro/samples-test.sh", ".kokoro/trampoline_v2.sh", ".github/release-trigger.yml", ".github/sync-repo-settings.yaml", ".kokoro/presubmit/node18/system-test.cfg"])
```

Comment:
Why are we skipping this ? If you have made changes in google3 , the changes should be allowed ?

Reply:
updated how we should skip backup tests